code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstanceStart.java

```diff
@@ -105,8 +105,9 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
                 // wait until volumesFrom/networksFrom containers start up
                 waitForDependenciesStart(instance);
 
+                progress.checkPoint("Waiting for deployment unit instances to allocate");
```

Comment:
@cjellick how about waitForDeploymentUnitCreate as a method name? just "wait" is a bit unclear